nengo/tests/test_builder.py

```diff
@@ -99,158 +99,6 @@ def create(make_extra, seed):
         assert same1seeds[same1obj] == same2seeds[same2obj]
 
 
-def test_signal():
```

Comment:
I swear I removed these in some other PR, but I don't remember where 🤷‍♂️ .  No harm in doing it twice though.